test/runtime/samples/css-vars-escape/_config.js

```diff
@@ -0,0 +1,12 @@
+export default {
+	html: `
+		<div style="display: contents; --color: &quot; onload=&quot;alert('uhoh')&quot; data-nothing=&quot;not important;">
+			<div class="svelte-1olrwfe">hi</div>
+		</div>
+	`,
+	ssrHtml: `
+		<div style="display: contents; --color:&quot; onload=&quot;alert('uhoh')&quot; data-nothing=&quot;not important;">
+			<div class="svelte-1olrwfe">hi</div>
+		</div>
+	`
+};
```

Comment:
This test is not failed even without this PR. But I couldn't find a test for `style props` escaping, so I added.